guide/C/gnucash-guide.xml

```diff
@@ -11,8 +11,10 @@
 <!-- List of files: -->
 <!ENTITY chapter1 SYSTEM "ch_oview.xml">
 <!ENTITY chapter2 SYSTEM "ch_basics.xml">
-<!ENTITY chapter3 SYSTEM "ch_accts.xml">
-<!ENTITY chapter4 SYSTEM "ch_txns.xml">
+<!ENTITY sectaccts SYSTEM "ch_accts.xml">
+<!ENTITY secttxns SYSTEM "ch_txns.xml">
+<!ENTITY chapter3 SYSTEM "ch_importing.xml">
+<!ENTITY chapter4 SYSTEM "ch_configuring.xml">
```

Comment:
1. How came the chapters to this place?
I seems, you used 'git add', but here the right way would be 'git mv ...' that would keep the history intact.
@gjanssens, @jralls: Do you know of a method to repair this issue without a complete redo?

2. I miss the (addition of the 2 new chapters to entities in guide/C/makefile.am and their removal from help/C/makefile.am)

IIRC, it was in the old autotool version of building page and got lost, when the program switched to CMake.

Reply:
Frank, I noted in the commit message that I added these chapters. As in, I created them with a text editor, saved them in my local repo, and added them to my fork. There is no other history. 

As for Makefile.am, when I looked at my local copy, I didn't see anything there that would need my change; it seemed to be a series of loops, but there were no files enumerated there.

Reply:
ad 1: I thought, it was some move of Help Chapter 3?

ad 2: gnucash-docs/guide/C/Makefile.am, not gnucash-docs/Makefile.am.

Reply:
1: My misunderstanding. What I did was to move the longer texts into the Guide, but I left the structural elements of Help in place and commented out the portions that had been moved to the Guide. In the course of this stripping, I found that the text from Help included references to sections that by rights should go into new chapters in the Guide, on Importing and Configuring. Since I was in the process already, I chose to add these two chapters as part of the PR.
2: I did not see THAT copy. My apologies. I have changed it now. Is it correct?

Reply:
Further: I did not remove any files from Help, just pared them down. So, no change to Makefile.am there.

Reply:
Changes implemented